src/pods/toolbar/toolbar.pod.tsx

```diff
@@ -31,14 +31,68 @@ export const ToolbarPod: React.FC = () => {
       />
     );
   };
-
   return (
     <div className={classes.container}>
-      <button onClick={zoomIn}>Zoom in</button>
-      <button onClick={zoomOut}>Zoom out</button>
-      <button onClick={handleRelationClick}>Relation</button>
-      <button onClick={handleEditTableClick}>Edit Table</button>
-      <button onClick={handleCanvasSettings}>Canvas Settings</button>
+      <h1>Toolbar</h1>
+      <div className={classes.buttonContent}>
```

Comment:
Toobar buttons should have the Icon and a text below (indication the action), could you create a toolbar button that encapsulates this functionallilty?